### DISCUSSION TITLE
My commit count not showing on Contribution graph

### DISCUSSION THREAD

**Discussion by ibrahimkholil6210** [community/community#22133] [New to GitHub]
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/4/41a94c7f7efbe9d3e92280eef36c055e948463b0.png" data-download-href="/uploads/short-url/9mRKUtU5ySU72XU3uBwb1gacZQA.png?dl=1" title="Capture" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/33357137/181069004-bfb28b04-eb25-4a43-aad0-ca76df872ff9.png" alt="Capture" data-base62-sha1="9mRKUtU5ySU72XU3uBwb1gacZQA" width="690" height="464" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/4/41a94c7f7efbe9d3e92280eef36c055e948463b0_2_690x464.png, https://aws1.discourse-cdn.com/github/optimized/2X/4/41a94c7f7efbe9d3e92280eef36c055e948463b0_2_1035x696.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/4/41a94c7f7efbe9d3e92280eef36c055e948463b0.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/4/41a94c7f7efbe9d3e92280eef36c055e948463b0_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Capture</span><span class="informations">1259×848 57.9 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div><br>
I am having trouble not getting to show my commit count in contribution graph section how to fix this issue

**Comment by airtower-luna** [comment, marked as answer]
<p>The default avatar there looks like the author email address in those commits is not on your account, so GitHub doesn’t recognize them as yours. There are two options to change that, depending on whether the email address in those commits is correct (as in, yours and able to receive mail):</p>
<ul>
<li>If it is correct, you can <a href="https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/adding-an-email-address-to-your-github-account" rel="nofollow noopener">add the mail address to your account</a>.</li>
<li>If it is not, you should <a href="https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/setting-your-commit-email-address" rel="nofollow noopener">change your commit email</a> to whichever is the right one. Note that that does not fix past commits. You can <a href="https://help.github.com/en/github/using-git/changing-author-info" rel="nofollow noopener">rewrite commit history to change that</a>, but read the warning at the top of that article and carefully consider if you really want to do that.</li>
</ul>

**Comment by ibrahimkholil6210** [comment]
<p>Thank you very much! i got it…</p>
